### PR TITLE
topolvm.io

### PR DESCRIPTION
**Pull request by bells17**
- #539
- #168

This PR was created by below commands:

```
git checkout topolvm.io
git diff main > /tmp/topolvm.io.patch
vi /tmp/topolvm.io.patch # Removed trailing whitespace.

git checkout main
git checkout -b topolvm.io3
git apply /tmp/topolvm.io.patch
```

Removed trailing whitespace:

```
git apply /tmp/topolvm.io.patch
/tmp/topolvm.io.patch:620: trailing whitespace.
{{- end }} 
/tmp/topolvm.io.patch:4646: trailing whitespace.
for each device-class and `capacity.topolvm.io/00default` annotation 
/tmp/topolvm.io.patch:4711: trailing whitespace.
To specify a device-class name to be used, give `topolvm.io/device-class` parameter. 
warning: 3 lines add whitespace errors.
```

diff: topolvm.io3 ~ topolvm.io

```
$ git diff topolvm.io
diff --git a/charts/topolvm/templates/_helpers.tpl b/charts/topolvm/templates/_helpers.tpl
index 2d54227d..2c4841e1 100644
--- a/charts/topolvm/templates/_helpers.tpl
+++ b/charts/topolvm/templates/_helpers.tpl
@@ -66,4 +66,4 @@ Plugin Name.
 */}}
 {{- define "topolvm.pluginName" -}}
 {{ if .Values.useLegacy }}topolvm.cybozu.com{{ else }}topolvm.io{{ end }}
-{{- end }} 
+{{- end }}
diff --git a/docs/topolvm-node.md b/docs/topolvm-node.md
index 99a6684b..4601a476 100644
--- a/docs/topolvm-node.md
+++ b/docs/topolvm-node.md
@@ -96,7 +96,7 @@ Node resource
 -------------
 
 `topolvm-node` adds `capacity.topolvm.io/<device-class>` annotations
-for each device-class and `capacity.topolvm.io/00default` annotation 
+for each device-class and `capacity.topolvm.io/00default` annotation
 for the default device-class to the corresponding `Node` resource of the running node.
 The value is the free storage capacity reported by `lvmd` in bytes.
 
diff --git a/docs/user-manual.md b/docs/user-manual.md
index 63528223..6a1ed01b 100644
--- a/docs/user-manual.md
+++ b/docs/user-manual.md
@@ -36,7 +36,7 @@ allowVolumeExpansion: true
 
 `parameters` are optional.
 To specify a filesystem type, give `csi.storage.k8s.io/fstype` parameter.
-To specify a device-class name to be used, give `topolvm.io/device-class` parameter. 
+To specify a device-class name to be used, give `topolvm.io/device-class` parameter.
 If no `topolvm.io/device-class` is specified, the default device-class is used.
 
 Supported filesystems are: `ext4` and `xfs`.
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,19 +7,6 @@ This file itself is based on [Keep a CHANGELOG](https://keepachangelog.com/en/0.
 
 ## [Unreleased]
 
-## [0.15.3] - 2022-11-04
```
Comment (peng225) [inline CHANGELOG.md:10]: These lines should not be deleted.

I think this is caused by the following steps.
```
git checkout topolvm.io
git diff main > /tmp/topolvm.io.patch
```

I guess the base of  `topolvm.io` branch is too old. You had to rebase it first as follows.
```
git checkout main
git pull
git checkout topolvm.io
git rebase main
git diff main > /tmp/topolvm.io.patch
```

**Review by nbalacha — COMMENTED** [review]
Some minor comments . I also see a mismatch between the number of files (107) in this and the original commit ( #539 has 103 files).

File: charts/topolvm/CHANGELOG.md [hunk 2]
```diff
@@ -7,20 +7,8 @@ This file itself is based on [Keep a CHANGELOG](https://keepachangelog.com/en/0.
 
 ## [Unreleased]
 
-## [10.0.0] - 2022-11-04
```
Comment (llamerada-jp) [inline charts/topolvm/CHANGELOG.md:10]: Changes to this file would not be required.

File: example/README.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ and loopback block devices to run `lvmd`.
 You can try TopoLVM using the example with tag like following command. The main branch is being edited, so it may not work.
 
 ```
-$ git checkout v0.15.3
```
Comment (llamerada-jp) [inline example/README.md:11]: ditto

**Review by llamerada-jp — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (bells17) [reply CHANGELOG.md:10]: Sorry, I'll fix it.

Comment (peng225) [reply CHANGELOG.md:10]: Thanks.
We need to move to the new PR https://github.com/topolvm/topolvm/pull/592 ?
It looks good.
After it become ready for review, I will review it.

**Comment by bells17** [comment]
@peng225 @llamerada-jp @nbalacha Sorry, I created a new PR. Would you reiview this?

https://github.com/topolvm/topolvm/pull/592

File: CHANGELOG.md (hunk 1 above)
Comment (bells17) [reply CHANGELOG.md:10]: > We need to move to the new PR https://github.com/topolvm/topolvm/pull/592 ?

Yes, please.

**Pull request closed without merging**